Services/DataX.Gateway/DataX.Gateway.Api/Controllers/GatewayController.cs

```diff
@@ -87,17 +88,29 @@ static GatewayController()
             var configPackage = FabricRuntime.GetActivationContext().GetConfigurationPackageObject("Config");
             var serviceEnvironmenConfig = configPackage.Settings.Sections["ServiceEnvironment"];
             var appInsightsIntrumentationKey = serviceEnvironmenConfig.Parameters["AppInsightsIntrumentationKey"].Value;
+            var testClientId = serviceEnvironmenConfig.Parameters["TestClientId"].Value;
```

Comment:
to be more generic, it is best to have a white list

Reply:
Done. Now the secret will be able to take in a ',' separated string of clientIds, essentially a list a unique Ids which would be a combination of  OId and TenantId.